docs/README.md

```diff
@@ -162,18 +165,64 @@ Sub-commands:
   * `4` or `full`: Currently the same as the `extended` summary, includes a list of all objects in the structure.
 * `-t`, `--traffic`: Log traffic information.
 * `-m`, `--trafficmetrics`: Log metrics information.
+* `-s`, `--set`: Override a value in script with key.path=value. See [Using script overrides](#using-script-overrides) for further explanation.
 
 `validate` command flags:
 
 * `-c`, `--config string`: Load the specified scenario setup file.
 * `-h`, `--help`: Show the help for the `validate` command.
+* `-s`, `--set`: Override a value in script with key.path=value. See [Using script overrides](#using-script-overrides) for further explanation.
 
 `template` command flags:
 
 * `-c`, `--config string`: (optional) Create the specified scenario setup file. Defaults to `template.json`.
 * `-f`, `--force`: Overwrite existing scenario setup file.
 * `-h`, `--help`: Show the help for the `template` command.
 
+#### Using script overrides
+
+Script overrides overrides a value pointed to by a path to its key. If the key doesn't exist in the script there will be a not found error, even if it's a valid value according to config.
```

Comment:
A script override overrides a value..
or
Script overrides override values..

Reply:
not-found-error? "not found"-error?

Reply:
> If the key doesn't exist in the script there will an error, even if it's a valid value according to config.
?